src/context/WalletProvider/KeepKey/hooks/useKeepKeyEventHandler.ts

```diff
@@ -114,6 +122,10 @@ export const useKeepKeyEventHandler = (
         case MessageType.PINMATRIXACK:
           if (modal) dispatch({ type: WalletActions.SET_WALLET_MODAL, payload: false })
           break
+        // A bit tricky but if we receive this event, the device is obviously not waiting for interaction
```

Comment:
This assertion is not true. We can absolutely receive a `ETHEREUMGETADDRESS` whilst waiting for a device interaction. 

Indeed, this seems to be causing a bug where the UI will think it's no longer awaiting a device interaction whilst the KeepKey is awaiting for PIN change confirmation.